index.html

```diff
@@ -63,5 +63,183 @@ <h1 class="narson">I'm Narson.
+            <ul class="languages-list">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="javascript">javascript</li>
+            </ul>
+            <button type="button" class="buttons">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!--About me section-->
+
+    <section class="about-me">
+        <div class="about-container">
+        <h3 class="about-text">About Myself</h3>
+        <p class="summary">
+            Hello I’m a software developer! I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <h4 class="connection2">LET'S CONNECT</h4>
+
+        <ul class="social2">
+            <li class="twitter">
+                <a href="https://twitter.com/@Narson321">
+                <img src="media/twitter.svg" alt="twitter" title="twitter"/>
+                </a>
+            </li>
+            <li class="linkindin">
+                <a href="https://linkedin.com/in/andry-narson-rabedesana-15b8b4248">
+                    <img src="media/Linkedin icon.svg" alt="Linkindin" title="linkindin"/>
+                </a>
+            </li>
+            <li class="github">
+                <a href="https://github.com/rbd3">
+                <img src="media/github.svg" alt="github" title="github" />
+                </a>
+            </li>
+            <li class="angellist"><a href="#">
+                </a>
+                <img src="media/angellist.svg" alt="Angellist" title="Angellist" />
+                </a>
+            </li>
+            <li class="medium">
+                <a href="#">
+                <img src="media/medium.svg" alt="Medium" title="Medium" />
+                </a>
+            </li>
+        </ul>
+        <button type="button" class="buttons2">Get my resume</button>
+        </div>
+
+        <div class="skill-list">
+            <p class="languages">Languages</p>
+            <img src="media/Icon - Chevron - Down - Larger.svg" alt="Down Arrow" class="down-arrow" />
+
+            <div class="languages-container">
+                <ul class="languages-list2">
+                    <li class="javascript2">
+                       <img class="js-logo" src="media/js.svg" alt="Javascript logo" />
+                       <p class="js-text">JavaScript</p> 
+                    </li>
+                    <li class="html2">
+                        <img class="html-logo" src="media/html.svg" alt="HTML logo" />
+                        <p class="html-text">HTML</p> 
+                     </li>
+                     <li class="css2">
+                        <img class="css-logo" src="media/css.svg" alt="CSS logo" />
+                        <p class="css-text">CSS</p> 
+                     </li>
+                </ul>
```

Comment:
- [ Required ] Good job trying to address the design of `Languages` section of your webpage as requested by the previous reviewer, however, while implementing the changes you introduced a margin on the `left-hand` side of your design that should not be there. Please note that the list items e.g `JavaScript` , `HTML`, and `CSS` should align on the same line with the letter `L` in `Languages`. Please see the screenshot below and rectify this.

## Your design
<img src="https://user-images.githubusercontent.com/16392046/235135125-da804962-774c-427d-81b7-5ac78c7ec3f5.png" width="375px" />

## Figma template
<img src="https://user-images.githubusercontent.com/16392046/235135513-c9772859-e5eb-4bec-9639-52fac7237d29.png" width="375px" />